pkg/volume/volume.go

```diff
@@ -111,13 +111,10 @@ type Recycler interface {
 // Provisioner is an interface that creates templates for PersistentVolumes and can create the volume
 // as a new resource in the infrastructure provider.
 type Provisioner interface {
-	// Provision creates the resource by allocating the underlying volume in a storage system.
-	// This method should block until completion.
-	Provision(*api.PersistentVolume) error
-	// NewPersistentVolumeTemplate creates a new PersistentVolume to be used as a template before saving.
-	// The provisioner will want to tweak its properties, assign correct annotations, etc.
-	// This func should *NOT* persist the PV in the API.  That is left to the caller.
-	NewPersistentVolumeTemplate() (*api.PersistentVolume, error)
```

Comment:
I merged these two methods to one, annotations should be there so is claimRef. Look at updated `Provision()` in each plugin.